pwndbg/commands/context.py

```diff
@@ -148,15 +148,16 @@ def get_regs(*regs):
 pwndbg.config.Parameter('emulate', True, '''
 Unicorn emulation of code near the current instruction
 ''')
+code_lines = pwndbg.config.Parameter('context-code-lines', 10, 'number of additional lines to print in the code context')
 
 def context_code():
     banner = [pwndbg.ui.banner("code")]
     emulate = bool(pwndbg.config.emulate)
-    result = pwndbg.commands.nearpc.nearpc(to_string=True, emulate=emulate)
+    result = pwndbg.commands.nearpc.nearpc(to_string=True, emulate=emulate, lines=int(int(code_lines) / 2))
```

Comment:
It may be desirable to extend the config class to have __div__ and __idiv__ and so-on methods to avoid the need to explicitly cast these